### ISSUE TITLE
Metadata with multiple options not saved without preset

### ISSUE THREAD

**Issue by solth** [kitodo/kitodo-production#4061]
Metadata defined by a `key` with multiple `option`s in the ruleset (resulting in a pulldown/selectOne menu) is can only be saved in the metadata editor if a `preset` is configured.

Metadata configured the following way **can** be saved:
```
<key id="metadata1">
	<label>Works</label>
	<option value="value1"/>
	<option value="value2"/>
	<option value="value3"/>
	<preset>value1</preset>
</key>
```

Metadata configured the following way **cannot** be saved:
```
<key id="metadata2">
	<label>Broken</label>
	<option value="value1"/>
	<option value="value2"/>
	<option value="value3"/>
</key>
```

**Comment by matthias-ronge** [comment]
I have now tested this with the `ruleset_test.xml`, which you made available to me yesterday, using the metadata `LegalNoteAndTermsOfUse`, which doesn’t declare a `<preset>`, on division `"Volume"`.

* with `minOccurs="1"` and `maxOccurs="1"`, `multiline="false"` is saved
* with `maxOccurs="1"`, `multiline="false"` is saved
* with `minOccurs="1"`, `multiline="false"` is saved
* unbounded, `multiline="false"` is saved
* unbounded, `multiline="false"` with `alwaysShowing="true"` is saved
* unbounded, `multiline="true"` with `alwaysShowing="true"` (list of checkboxes) is **not saved**
* with `maxOccurs="1"`, `multiline="true"` (list of radio buttons) is **not saved**
* with `maxOccurs="1"`, without any `<setting>` (drop-down select) is saved
* with `minOccurs="1"` and `maxOccurs="1"`, without any `<setting>` (drop-down select) is saved. Btw, the select box offers „no selection“ (seems to be added by the interface) and an empty entry (added by the ruleset management). Only one of the two is necessary.

**Comment by matthias-ronge** [comment]
Just to be sure, I also tested with `"metadata2"` as you indicated above (no `<label>` elements), but it also saves.

**Comment by matthias-ronge** [comment]
Today I encounter the situation that with build of 37d56a0c5965c49efba3b6d5df5fef102737f8a0 the constellation `minOccurs="1"` and `maxOccurs="1"` is **not saved**, neither with explicit `multiline="false"` nor without `<setting>` at all.

**Comment by BartChris** [comment]
I tried to reproduce that, but failed so far on current Main. Is this really still bugged in current main and is this still the current precise issue?

```xml
<key id="metadata2">
	<label>Broken</label>
	<option value="value1"/>
	<option value="value2"/>
	<option value="value3"/>
</key>
```

> with minOccurs="1" and maxOccurs="1", multiline="false" is not saved

**Comment by solth** [comment]
> I tried to reproduce that, but failed so far on current Main. Is this really still bugged in current main and is this still the current precise issue?

Yes, you are right, I cannot reproduce this particular problem in the current `main` branch either, so I will close the issue. There are, however, still other problems with saving metadata. For example, boolean metadata that is rendered via check boxes does not get saved correctly. See #5689 for details.

Ideally, we should have tests for _all_ the cases @matthias-ronge listed above, to ensure this central part of the application works as intended.
